services/movie.py

```diff
@@ -0,0 +1,41 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: int = None,
+        actors_ids: int = None

```

Comment:
If argument can be `None` you should show it in type hint. Fix everywhere
```suggestion
        genres_ids: int | None = None,
        actors_ids: int | None = None

```